packages/schematics/src/cypress/rules/verify-options.ts

```diff
@@ -0,0 +1,45 @@
+import { SchematicsException, Tree } from '@angular-devkit/schematics';
+import { CypressSchematicOptions } from '../model';
+import { getE2eProjectNameForApp, readAngularJson } from './utils';
+
+export function verifyOptions(
+  options: CypressSchematicOptions,
+  tree: Tree
+): CypressSchematicOptions {
+  const angularJson = readAngularJson(tree);
```

Comment:
maybe rename it to `project`

Reply:
But angular.json contains multiple projects. Maybe the `app` is more like the `projectName`?. But I can see, that there are some issues with naming.

Reply:
Yeah, you are right.
I came up with `project` since this is the common wording in the `.NET World`.
There is a `.csproj`-file defining a project.
You can combine multiple projects using a `.sln`-file (solution).

But in our case project, does not make sense any more.

Maybe `ngConfig` fits better. 🤷‍♂️ 😉

Reply:
done :-)